### PR TITLE
Fixing TabNet sparsity loss issue

### PR DESCRIPTION
**Pull request by w4nderlust**

### CODE REVIEW

**Review by tgaddair — APPROVED** [review]

File: ludwig/models/ecd.py [hunk 1]
```diff
@@ -209,8 +209,17 @@ def train_loss(self, targets, predictions, regularization_lambda=0.0):
                                               predictions[of_name])
             train_loss += of_obj.loss['weight'] * of_train_loss
             of_train_losses[of_name] = of_train_loss
-        train_loss += regularization_lambda * sum(
-            self.losses)  # regularization / other losses
+
+        for loss in self.losses:
+            if hasattr(loss, "loss_name"):
```
Comment (tgaddair) [inline ludwig/models/ecd.py:214]: Is there a more consistent way it check this with an `isinstance` condition or similar?
